components/gauge-circular/scale.md

```diff
@@ -0,0 +1,200 @@
+
+>caption Change the rendering of the minor and major unit divisions. The result from the code snippet below.
+
+![Minor and major units parameters](images/minor-and-major-units-circular-gauge.png)
+
+````CSHTML
+@* Update the rendering of the major and minor units. *@
+
+<TelerikCircularGauge>
+
+    <CircularGaugeScales>
+
+        <CircularGaugeScale MajorUnit="10" MinorUnit="5">
+            <CircularGaugeScaleMajorTicks Visible="true"></CircularGaugeScaleMajorTicks>
+            <CircularGaugeScaleMinorTicks Visible="true"></CircularGaugeScaleMinorTicks>
+            <CircularGaugeScaleLabels Visible="true"></CircularGaugeScaleLabels>
+        </CircularGaugeScale>
+
+    </CircularGaugeScales>
+
+    <CircularGaugePointers>
+
+        <CircularGaugePointer Value="70">
+        </CircularGaugePointer>
+
+    </CircularGaugePointers>
+</TelerikCircularGauge>
+````
+
+## Start Angle
+
+The `StartAngle` (`double`) parameter controls the start angle of the gauge. The gauge is rendered clockwise (0 degrees are the 180 degrees in the polar coordinate system). Defaults to 90 degrees.
+
+>caption Change the Start Angle of the Circular Gauge. The result from the code snippet below.
+
+![Start Angle parameter](images/start-angle-circular-gauge.png)
+
+````CSHTML
+@* Use StartAngle parameter to change the start angle of the scale. *@
+
+<TelerikCircularGauge>
+    <CircularGaugeScales>
+
+        <CircularGaugeScale StartAngle="0">
+            <CircularGaugeScaleMajorTicks Visible="true"></CircularGaugeScaleMajorTicks>
+            <CircularGaugeScaleMinorTicks Visible="true"></CircularGaugeScaleMinorTicks>
+            <CircularGaugeScaleLabels Visible="true"></CircularGaugeScaleLabels>
+        </CircularGaugeScale>
+
+    </CircularGaugeScales>
+
+    <CircularGaugePointers>
+
+        <CircularGaugePointer Value="60">
+        </CircularGaugePointer>
+
+    </CircularGaugePointers>
+</TelerikCircularGauge>
+````
+
+## Reverse
+
+If you set the `Reverse` (`bool`) parameter to `true` the scale direction will be reversed - values will increase anticlockwise.
+
+>caption Reverse the scale of the Circular Gauge. The result from the code snippet below.
+
+![Reverse parameter example](images/reverse-circular-gauge.png)
+
+````CSHTML
+@* Reverse the scale. *@
+
+<TelerikCircularGauge>
+    <CircularGaugeScales>
+
+        <CircularGaugeScale Reverse="true">
+            <CircularGaugeScaleMajorTicks Visible="true"></CircularGaugeScaleMajorTicks>
+            <CircularGaugeScaleMinorTicks Visible="true"></CircularGaugeScaleMinorTicks>
+            <CircularGaugeScaleLabels Visible="true"></CircularGaugeScaleLabels>
+        </CircularGaugeScale>
+
+    </CircularGaugeScales>
+
+    <CircularGaugePointers>
+
+        <CircularGaugePointer Value="60">
+        </CircularGaugePointer>
+
+    </CircularGaugePointers>
+</TelerikCircularGauge>
+````
+
+## Additional Customization
```

Comment:
The section renders empty, maybe the template is missing

Reply:
Done